.github/workflows/test-template.yml

```diff
@@ -316,7 +316,7 @@ jobs:
           engine-hash: ${{ inputs.engineHash }}
           skip-tsc: false
 
-      - run: pnpm run test:functional --flavor ${{ matrix.flavor }} --shard ${{ matrix.shard }} --engine-type ${{ matrix.engine-type }} --runInBand
+      - run: pnpm run test:functional --flavor ${{ matrix.flavor }} --shard ${{ matrix.shard }} --engine-type ${{ matrix.engine-type }} --preview-features driverAdapters --runInBand
```

Comment:
Does this _add_ another run with preview feature `driverAdapters` or does this add the preview flag to this run and change what is actually tests?

Reply:
It is not a separate run or test suite name, and it is completely unknown to the tests. This is the same run as before and we now inject the preview feature via the CLI direct instead of via previously [hardcoded conditions](https://github.com/prisma/prisma/pull/22246/files#diff-1d8cafa403bb80bfd8c3562210e02f7ffd423c74e5a9064984a9b0e79e48ce06L178).